src/Compiler/Microsoft.AspNetCore.Razor.Language/src/CommonMetadata.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Razor.Language.Components;
+
+namespace Microsoft.AspNetCore.Razor.Language;
+
+public static class CommonMetadata
+{
+    internal static readonly KeyValuePair<string, string?> BindAttributeGetSet
+        = IsTrue(ComponentMetadata.Bind.BindAttributeGetSet);
+    internal static readonly KeyValuePair<string, string?> IsDirectiveAttribute
+        = IsTrue(ComponentMetadata.Common.DirectiveAttribute);
+    internal static readonly KeyValuePair<string, string?> IsWeaklyTyped
+        = IsTrue(ComponentMetadata.Component.WeaklyTypedKey);
+
+    internal static KeyValuePair<string, string?> IsTrue(string key)
```

Comment:
Nit: this naming super confused me for a second. Consider a name like `MakeTrue`.

Reply:
Good idea. Will do.